content-src/asrouter/schemas/provider-response.schema.json

```diff
@@ -2,7 +2,7 @@
   "title": "ProviderResponse",
   "description": "A response object for remote providers of AS Router",
   "type": "object",
-  "version": "0.1.0",
+  "version": "6",
```

Comment:
Is this to line up with the services endpoint?

Reply:
Yes, we were using that version number to format the URL pref